labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -25,7 +25,11 @@ object OptionExercise01 {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    rooms.get(room) match {
+      case Some(Some(string)) => if(string == "locked") "not available" else string
```

Comment:
I also used a lot of ternary operators in my pattern matching cases. Made things a lot easier! Don't know if it's Scala-y. (A lot of these comments seem to have the reoccurring theme of me not knowing whether things are Scala-y)